src/Constants.h

```diff
@@ -0,0 +1,12 @@
+//Menu-Message-ID
+const uint32 MENU_APP_QUIT = 1000;
```

Comment:
Actually I found the constants declared in the codebase for most of the apps in separate files so I thought it's preferred to keep it this way. The scope should be the narrowest one anyway. Do I change the same for other apps in there?

Okay, I'll rename them. I thought about doing the same but found it that way in almost all repos so restricted myself.